apps/src/componentLibrary/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -44,6 +44,7 @@ const Breadcrumbs: React.FunctionComponent<BreadcrumbsProps> = ({
         moduleStyles[`breadcrumbs-${size}`],
         className
       )}
+      // eslint-disable-next-line react/forbid-dom-props
```

Comment:
@levadadenys any input since this change is being applied to DSCO components?

Reply:
Thanks for pinging me!
Yep, LGTM so far. Not sure if it'll be possible to fix all the existing data-testId usages, but I do think it's possible to at least decrease number of data-testId usages.
Created a tickets which also includes checking this eslint-disable statements